app/pb/events.proto

```diff
@@ -0,0 +1,304 @@
+    // Schedules a new occurence of an existing event
+  }
+
+  rpc UpdateEvent(UpdateEventReq) returns (Event) {
+    // Edit an event
+  }
+
+  rpc GetEvent(GetEventReq) returns (Event) {
+    // Gets an event
+  }
+
+  rpc ListEventOccurences(ListEventOccurencesReq) returns (ListEventOccurencesRes) {
+    // List all occurences of a recurring event
+  }
+
+  rpc ListEventAttendees(ListEventAttendeesReq) returns (ListEventAttendeesRes) {
+    // Paginated list of event attendees
+  }
+
+  rpc ListEventSubscribers(ListEventSubscribersReq) returns (ListEventSubscribersRes) {
+    // Paginated list of event subscribers
+  }
+
+  rpc ListEventOrganizers(ListEventOrganizersReq) returns (ListEventOrganizersRes) {
+    // Paginated list of event organizers
+  }
+
+  rpc TransferEvent(TransferEventReq) returns (Event) {
+    // Transfer an event you own to either a group or a community
+  }
+
+  rpc SetEventSubscription(SetEventSubscriptionReq) returns (Event) {
+    // Subscribe to get notifications about all occurences of an event
+  }
+
+  rpc SetEventAttendance(SetEventAttendanceReq) returns (Event) {
+    // RSVP/change going status for an occurence of an event
+  }
+
+  rpc InviteEventOrganizer(InviteEventOrganizerReq) returns (google.protobuf.Empty) {
+    // Invite another user to become an organizer of this event
+  }
+
+  rpc RemoveEventOrganizer(RemoveEventOrganizerReq) returns (google.protobuf.Empty) {
+    // Remove yourself as an event organizer
+  }
+
+  rpc ListMyEvents(ListMyEventsReq) returns (ListMyEventsRes) {
+    // Lists all subscribed, attending
+  }
+
+  // TODO: DownloadCalendarInvite
+}
+
+message OnlineEventInformation {
+  // videoconferencing link, etc
+  string link = 1;
+}
+
+message OfflineEventInformation {
+  // in-person events must have address + location
+  string address = 1;
+  double lat = 2;
+  double lng = 3;
+}
+
+enum AttendanceState {
+  ATTENDANCE_STATE_NOT_GOING = 0;
+  ATTENDANCE_STATE_MAYBE = 1;
+  ATTENDANCE_STATE_GOING = 2;
+}
+
+message Event {
+  int64 event_id = 1; // = occurence_id
+  // it's the next event occurence
+  bool is_next = 2;
+  // already happened
+  bool is_past = 3;
+  // in the future
+  bool is_future = 4;
+
+  string title = 5;
+  // a short URL fragment generated from the name, e.g. "New York City: weekly meetup" might become "new-york-city-weekly-meetup"
+  string slug = 6;
+  string content = 7;  // CommonMark without images
+  // empty string if no photo
+  string photo_url = 8;
+
+  oneof mode {
+    OnlineEventInformation online_information = 9;
+    OfflineEventInformation offline_information = 10;
+  }
+
+  google.protobuf.Timestamp created = 13;
+  google.protobuf.Timestamp last_edited = 14;
+  int64 creator_user_id = 15;
+
+  // the UTC timestamps of start and end times
+  google.protobuf.Timestamp start_time = 16;
+  google.protobuf.Timestamp end_time = 17;
+
+  // the tzdata timezone identified
+  string timezone = 18;
```

Comment:
identified? I think this is the timezone decided when creating the event.

Reply:
It's a typo, should be "identifier"